HARK/ConsumptionSaving/ConsRiskyAssetModel.py

```diff
@@ -239,9 +357,919 @@ def get_shocks(self):
         self.get_Adjust()
 
 
-# %% Initial parameter sets
+# This is to preserve compatibility with old name
+RiskyAssetConsumerType = IndShockRiskyAssetConsumerType
+
+
+class FixedPortfolioShareRiskyAssetConsumerType(IndShockRiskyAssetConsumerType):
```

Comment:
It strikes me as odd that:
 - IndShockRiskyAssetConsumerType has two 'modes' : strategic share variation, or with a fixed rate at 1
 -  FixedPortfolioShareRiskyAssetConsumerType is subclass of the above class, with a variable fixed rate.

If I'm not mistaken, the fixed-rate-of-1 agent is a special case of the fixed-rate agent, and so logically, the former should be a subclass, if not a specific parameterization, of the latter

Also...these class names are getting to be a little ridiculous. There has got to be a way to shorten them. This may entail a larger conversation about model class naming conventions.